x-pack/plugins/actions/server/lib/get_oauth_access_token.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { Logger } from '@kbn/core/server';
+import { GoogleAuth } from 'google-auth-library';
+import { ConnectorToken, ConnectorTokenClientContract } from '../types';
+
+interface GetOAuthJwtAccessTokenOpts {
+  connectorId?: string;
+  logger: Logger;
+  credentials: string;
+  connectorTokenClient?: ConnectorTokenClientContract;
+}
+
+export const getOAuthJwtAccessToken = async ({
```

Comment:
Could you please rename this function and the file to specify it is for google?
```suggestion
export const getGoogleOAuthAccessToken = async ({
```

Reply:
Required changes has been done.